.github/workflows/build.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         with:
           fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
       - name: Install sonar-scanner and build-wrapper
-        uses: SonarSource/sonarcloud-github-c-cpp-addition@v1
+        uses: SonarSource/sonarcloud-github-c-cpp@v1
```

Comment:
could you also update the README with the link to the GitHub action docs? Similarly, as I did e.g. [here](https://github.com/sonarsource-cfamily-examples/macos-xcode-gh-actions-sc/pull/11)? :)

Reply:
sure, done :)